Jenkinsfile

```diff
@@ -174,6 +174,25 @@ pipeline {
             echo 'Skipping Kafka Streams archetype test for Java 21'
           }
         }
+
+        stage('PowerPC') {
+          agent { label 'ppc64le' }
+          tools {
+            jdk 'jdk_21_latest'
+          }
+          options {
+            timeout(time: 8, unit: 'HOURS')
+            timestamps()
+          }
+          environment {
+            SCALA_VERSION=2.13
+          }
+          steps {
+            doValidation()
+            doTest(env, 'unitTest')
```

Comment:
The KIP does not state that only `unitTest` would run. Why are we doing setting this?

Reply:
Corrected to `doTest(env)`, missed to remove it.